Assets/Scripts/Tank/Enemy/EnemyController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using UnityEngine;
+[RequireComponent(typeof(Animator))]
+public class EnemyController : MonoBehaviour, Idamagable, IAttacker
+{
+    [SerializeField]
+    private float speed = 0.8f;
+    public float Speed{
+        set{
+            speed = value;
+        }
+    }
+
+    [SerializeField]
+    private float thrust = 0.5f;
+    public float Thrust{
+        set{
+            thrust = value;
+        }
+    }
+
+    [SerializeField]
+    private int health = 100;
+    public int Health{
+        set{
+            health = value;
+        }
+    }
+
+    private TankService tankService;
```

Comment:
Why do you need to store this in a variable?

Reply:
removed.